# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/WSRunObject.h
# Commit: 54a21814751f
# Full Hash: 54a21814751f05d88f3b9cf5a16a821d11ca56eb
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2020-03-02 21:27:32
# Regressor Bug: 1618089
# File Overlap Count: 2
# Description:
#   Bug 1618089 - part 2: Rename `WSRunScanner::Get(Next|Previous)CharPointInternal()` to `WSRunScanner::LookFor(Next|Previous)CharPointWithinAllTextNodes()` r=m_kato
#   
#   They are fallback methods when the container of given point is not a text node
#   in `WSRunScanner::mNodeArray`, and then, looks for a text node for the given
#   point.
# ==============================================================================

diff -r 4b43fd9bf671 -r 54a21814751f editor/libeditor/WSRunObject.h
--- a/editor/libeditor/WSRunObject.h	Mon Mar 02 07:33:23 2020 +0000
+++ b/editor/libeditor/WSRunObject.h	Mon Mar 02 07:33:56 2020 +0000
@@ -531,19 +531,19 @@
   WSPoint GetNextCharPoint(const WSPoint& aPoint) const;
 
   /**
-   * GetNextCharPointInternal() and GetPreviousCharPointInternal() are
-   * helper methods of GetNextCharPoint(const EditorRawDOMPoint&) and
-   * GetPreviousCharPoint(const EditorRawDOMPoint&).  When the container
-   * isn't in mNodeArray, they call one of these methods.  Then, these
-   * methods look for nearest text node in mNodeArray from aPoint.
-   * Then, will call GetNextCharPoint(const WSPoint&) or
+   * LookForNextCharPointWithinAllTextNodes() and
+   * LookForPreviousCharPointWithinAllTextNodes() are helper methods of
+   * GetNextCharPoint(const EditorRawDOMPoint&) and GetPreviousCharPoint(const
+   * EditorRawDOMPoint&).  When the container isn't in mNodeArray, they call one
+   * of these methods.  Then, these methods look for nearest text node in
+   * mNodeArray from aPoint. Then, will call GetNextCharPoint(const WSPoint&) or
    * GetPreviousCharPoint(const WSPoint&) and returns its result.
    */
   template <typename PT, typename CT>
-  WSPoint GetNextCharPointInternal(
+  WSPoint LookForNextCharPointWithinAllTextNodes(
       const EditorDOMPointBase<PT, CT>& aPoint) const;
   template <typename PT, typename CT>
-  WSPoint GetPreviousCharPointInternal(
+  WSPoint LookForPreviousCharPointWithinAllTextNodes(
       const EditorDOMPointBase<PT, CT>& aPoint) const;
 
   nsresult GetWSNodes();
